tests/cupyx_tests/scipy_tests/special_tests/test_gammainc.py

```diff
@@ -84,6 +84,10 @@ def gammainc_line(self, x):
     #     dataset = cp.vstack((a, x, self.gammainc_line(x))).T
     #     FuncData(sc.gammainc, dataset, (0, 1), 2, rtol=1e-11).check()
 
+    @pytest.mark.skipif(
+        cp.cuda.runtime.is_hip and
+        cp.cuda.runtime.runtimeGetVersion() < 5_00_00000,
+        reason='ROCm/HIP fails in ROCm 4.x')
```

Comment:
I just tested this with rocm 5.0.1 and almost all the tests fails :(.
Maybe we should skip the whole gammainc tests for hip ...

Reply:
Oh, when I tested this passed with ROCm 5.0.1. (note that you will need #6466 for ROCm 5.0)

Reply:
I just checked with the atomic header changes. I will re-run with #6466 fully applied! thanks.

However, the test that causes 4.3 to crash is a different one :)

Reply:
master with that branch applied:
```
tests/cupyx_tests/scipy_tests/special_tests/test_gammainc.py .......F.FFF.F.F.......FFFFF
```

Reply:
worked after clearing cache and compile from 0 everything

Reply:
List of tests that are broken in rocm 4.3
```
TestGammainc::test_domain[nan-1]
TestGammainc::test_infinite_limits
TestGammainc::test_limit_check
TestGammainc::test_roundtrip
TestGammaincc::test_infinite_limits
TestGammaincc::test_roundtrip
```
These work in rocm 5.0.1 so should be skipped too :)